src/components/Tabset/__test__/tabset.spec.js

```diff
@@ -14,6 +31,22 @@ describe('<Tabset />', () => {
         );
         expect(component.find('p').text()).toBe('testing tabset');
     });
+    it('should have the right class names', () => {
+        const component = mount(
+            <Tabset className="my-custom-class-name">
+                <Tab label="Tab-1" name="tab-1" registerTab={registerTabMockFn} />
+            </Tabset>,
+        );
+        expect(component.find('div').prop('className')).toBe('rainbow-tabset my-custom-class-name');
+    });
+    it('should have the right class names when fullWidth prop is passed', () => {
+        const component = mount(
+            <Tabset fullWidth>
+                <Tab label="Tab-1" name="tab-1" registerTab={registerTabMockFn} />
+            </Tabset>,
+        );
+        expect(component.find('ul').prop('className')).toBe('rainbow-tabset_inner-container rainbow-tabset_inner-container--full-width');
```

Comment:
in this case it is right because is very difficult that another ul gets added but you can use:
```ul[role="tablist"]```